.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MESSAGES CONTROL]
+
+disable=broad-except,
+        too-few-public-methods,
+        parse-error,
+        invalid-name,
+        attribute-defined-outside-init,
+        pointless-statement,
+        useless-object-inheritance,
+        no-else-return,
+        no-name-in-module,
+        len-as-condition,
+        missing-docstring,
+        bad-whitespace,
+        duplicate-code,
+        unused-import,
+        unused-variable,
+        useless-else-on-loop,
+        bad-continuation,
+        import-error,
+        too-many-arguments,
+        logging-not-lazy,
+        undefined-variable,
+        syntax-error,
```

Comment:
This is only needed, because `LICENSE`, `MANIFEST.in`, `PUBLISH.md` and `README.md` are linted.
```
LICENSE:1:0: E0001: unexpected indent (<unknown>, line 1) (syntax-error)
MANIFEST.in:1:0: E0001: invalid syntax (<unknown>, line 1) (syntax-error)
PUBLISH.md:2:0: E0001: invalid syntax (<unknown>, line 2) (syntax-error)
README.md:1:0: E0001: invalid syntax (<unknown>, line 1) (syntax-error)
```